server/router/src/main/java/org/infinispan/server/router/router/impl/rest/RestRouter.java

```diff
@@ -0,0 +1,93 @@
+package org.infinispan.server.router.router.impl.rest;
+
+import org.infinispan.commons.logging.LogFactory;
+import org.infinispan.rest.Server;
+import org.infinispan.rest.logging.RestAccessLoggingHandler;
+import org.infinispan.server.router.RoutingTable;
+import org.infinispan.server.router.configuration.RestRouterConfiguration;
+import org.infinispan.server.router.logging.RouterLogger;
+import org.infinispan.server.router.router.Router;
+import org.infinispan.server.router.routes.PrefixedRouteSource;
+import org.infinispan.server.router.routes.rest.NettyRestServerRouteDestination;
+import org.jboss.resteasy.plugins.server.netty.NettyJaxrsServer;
+import org.jboss.resteasy.spi.ResteasyDeployment;
+
+import javax.ws.rs.container.ContainerRequestFilter;
+import javax.ws.rs.container.ContainerResponseFilter;
+import java.lang.invoke.MethodHandles;
+import java.net.InetAddress;
+import java.util.Optional;
+
+public class RestRouter implements Router {
+
+    private static final RouterLogger logger = LogFactory.getLog(MethodHandles.lookup().lookupClass(), RouterLogger.class);
```

Comment:
As I mentioned before - I would prefer to leave it but if you insist, I'll change it.